.travis.yml

```diff
@@ -0,0 +1,18 @@
+# from http://jekyllrb.com/docs/continuous-integration/
+language: ruby
+rvm:
+- 2.1
+- 2.2
+# Assume bundler is being used, therefore
+# the `install` step will run `bundle install` by default.
+install: gem install jekyll rdiscount pygments.rb #html-proofer
+script: jekyll build #&& htmlproof ./_site --disable-external --only-4xx --check-html --check-favicon
+# branch whitelist, only for GitHub Pages
+branches:
+  only:
+  - gh-pages     # test the gh-pages branch
```

Comment:
Probably needs to be `master`, since this is a special repo where the master branch is the site (I think)